backend-api/src/functions/asyncBiometricToken/asyncBiometricTokenHandler.ts

```diff
@@ -217,18 +242,14 @@ async function handleUpdateSessionError(
   switch (updateSessionResult.value.errorType) {
     case UpdateSessionError.CONDITIONAL_CHECK_FAILURE:
       sessionAttributes = updateSessionResult.value.attributes;
-      return handleUpdateSessionConditionalCheckFailure(
+      return handleUnauthorizedConditionalCheckFailure(
```

Comment:
```suggestion
      return handleConditionalCheckFailure(
```

I don't think we should couple the output of the failure we're handling with the cause of the failure

Reply:
Updated and the other one that's like this